### PR TITLE
added legal actions in a belief, added test related to new function

### LINKED ISSUE #27: Missing legal actions with respect to a Belief
When the actions function is called on a belief, the environment will return all the actions instead of only the legal ones.

### PR DESCRIPTION
**Pull request by LorenzoBonanni**
fixed #27

### CODE REVIEW

File: src/actions.jl [hunk 1]
```diff
@@ -23,3 +23,7 @@ function POMDPs.actions(pomdp::RockSamplePOMDP{K}, s::RSState) where K
     end
 end
 
+function POMDPs.actions(pomdp::RockSamplePOMDP{K}, b::ParticleCollection) where K
+    state = rand(Random.GLOBAL_RNG, b)
```
Comment (zsunberg) [inline src/actions.jl]:
```suggestion
    # All states in a belief should have the same position, which is what the valid action space depends on
    state = rand(Random.GLOBAL_RNG, b) 
```

**Review by zsunberg — CHANGES_REQUESTED** [review]
@LorenzoBonanni Thanks for your contribution. I think it is worth adding a comment that I suggested. If you agree, I'll merge this.

**Comment by LorenzoBonanni** [comment]
@zsunberg the comment is ok, you can merge.
I thought about it but I didn't wrote anything since the function is specific about the environment and in this case the position is the same across all the belief.

**Review by zsunberg — APPROVED** [review]
Looks good! thanks for adding tests and making a nice PR! I'll merge shortly.

**Pull request merged**